components/br-stream/src/endpoint.rs

```diff
@@ -104,18 +104,19 @@ where
         scheduler: Scheduler<Task>,
     ) -> Result<()> {
         let tasks = meta_client.get_tasks().await?;
-        let mut watcher = meta_client.events_from(tasks.revision).await?;
         for task in tasks.inner {
-            info!("starts watch task {:?} from backup stream", task);
+            info!("watch task"; "task" => ?task);
```

Comment:
I would like add a common word for all br-stream module logs. cause tikv may generate too many debug logs. perhaps we could use `less tikv.log | grep {common word}` to obtain the related logs.

backup stream is the common word before.